src/nxdoc/nuxeo-add-ons/nuxeo-coldstorage/nuxeo-coldstorage-functional-overview.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Functional Overview
+description: Discover Nuxeo coldstorage capabilities.
+review:
+    comment: ''
+    date: ''
+    status: ''
+labels:
+    - coldstorage
+    - glacier
+    - storage
+toc: true
+tree_item_index: 100
+---
+
+## Coldstorage
```

Comment:
@jaubenque can you confirm if Coldstorage should be used as a single word or 2 (Cold Storage) please?

Reply:
I confirmed with Julien it should be Cold Storage and corrected all the instances